package/python-kiwi-spec-template

```diff
@@ -55,13 +55,11 @@ License:        GPL-3.0-or-later
 Packager:       Marcus Schaefer <marcus.schaefer@suse.com>
 %endif
 Group:          %{pygroup}
-Source:         %{name}.tar.gz
+Source0:        %{name}.tar.gz
 Source1:        %{name}-rpmlintrc
-# PATCH-FIX-OPENSUSE kiwi-revert-bls-default-for-suse.patch
-# Temporarily revert grub-bls default to false for SUSE distributions
-%if 0%{?suse_version}
-Patch:          kiwi-revert-bls-default-for-suse.patch
-%endif
+# SUSE-specific patches (1001+)
+## PATCH-FIX-OPENSUSE kiwi-revert-bls-default-for-suse.patch -- temporary until opensuse has bls
```

Comment:
Not sure that openSUSE nor SUSE will have BLS generally ...

Reply:
It'll happen. If for nothing else, because we in kiwi will push for it harder as we don't want to support more weird ways to do things.